pytorch_lightning/utilities/reproducibility.py

```diff
@@ -0,0 +1,21 @@
+"""Helper functions to help with reproducibility of models """
+import os
+
+import numpy as np
+import random
+import torch
+
+
+def seed_everything(seed: int = None):
+    """Function that sets seed for pseudo-random number generators  """
+    try:
+        seed = int(seed)
+    except (TypeError, ValueError):
+        return
```

Comment:
we should issue a warning so the user is aware a seed was not actually set

Reply:
I moved to `pytorch_lightning/trainer/seed.py` not sure if that is a good idea though. It is just one function, maybe it is better to place it somewhere in existing module